src-java/kilda-model/src/main/java/org/openkilda/model/Metadata.java

```diff
@@ -15,34 +15,43 @@
 
 package org.openkilda.model;
 
-import lombok.Value;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
 
 import java.io.Serializable;
 
 /**
  * Represents information about a metadata.
- * Uses 64 bit to encode information about the packet:
- *  0                   1                   2                   3
- *  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ * Uses 32 bit to encode information about the packet:
+ * 0                   1                   2                   3
+ * 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
  * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
- * |L|O|                      Reserved Prefix                      |
- * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
- * |                          Reserved Prefix                      |
+ * |L|O|A                       Reserved Prefix                      |
```

Comment:
1. missed `|` after `A`
2. there is no information about encapsulation ID mask here
3. placement of flags is wrong. Now flags are in the middle. Between prefix and encapsulationId